taichi/ir/statements.h

```diff
@@ -535,7 +576,9 @@ class AssertStmt : public Stmt {
 /**
  * Call an external (C++) function.
  */
-class ExternalFuncCallStmt : public Stmt {
+class ExternalFuncCallStmt : public Stmt,
```

Comment:
I wonder if concept would also allow us to avoid multi-inheritance..

Reply:
I think it should? Since these "Traits" are tied to actual C++ types (not runtime)